app/src/main/res/values/strings.xml

```diff
@@ -1215,4 +1215,8 @@
     <string name="certificate_info_verified_by">Verified By: %1$s </string>
     <!-- Login overflow menu delete button -->
     <string name="login_menu_delete_button">Delete</string>
+    <!-- The title of the Wikipedia top site -->
+    <string name="default_top_site_wikipedia">Wikipedia</string>
```

Comment:
I'm not sure about this :/ I think maybe "YouTube" is the same in every locale, but "Wikipedia" is localized? ie see jp.wikipedia.org (entirely in Japanese) or ru.wikipedia.org (in Cyrillic)
Looking for other thoughts

Reply:
CC @boek would love ur 🧠 on this

Reply:
I'll try to dig up an answer, but for now let's not localize these names because they are trademark/brand names so we don't have to block this PR landing.